server/gfmd/db_journal_apply.c

```diff
@@ -178,6 +178,40 @@ db_journal_apply_user_remove(gfarm_uint64_t seqnum, char *username)
 	return (e);
 }
 
+/**********************************************************/
+/* user_auth */
+
+static gfarm_error_t
+db_journal_apply_user_auth_add_or_modify(gfarm_uint64_t seqnum,
+	struct db_user_auth_arg *arg)
+{
+	gfarm_error_t e;
+	struct user *u;
+
+	u = user_tenant_lookup(arg->username);
+	if (u == NULL)
+		return (GFARM_ERR_NO_SUCH_USER);
+
+	e = user_auth_id_moidfy(u, arg->auth_method, arg->auth_user_id);
```

Comment:
there is a typo here (due to my fault).
Please replace user_auth_id_moidfy by user_auth_id_modify.